assets/javascript/script.js

```diff
@@ -178,6 +200,17 @@ $(document).ready(function () {
             let results = response;
             console.log(results);
 
+<<<<<<< HEAD
```

Comment:
It looks like you committed a file with a merge conflict

Reply:
Odd thing is github is not saying it is a merge conflict...there would be a resolution button if so

Reply:
Jason said this means that a merge marker got committed to the source code and that we should fix it.  No idea how though.  Going to have to look it up.